lang/cs/Org.Apache.REEF.IO/PartitionedData/FileSystem/FileSystemInputPartitionConfiguration.cs

```diff
@@ -39,6 +39,11 @@ public sealed class FileSystemInputPartitionConfiguration<T> : ConfigurationModu
         public static readonly RequiredParameter<string> FileSerializerConfig = new RequiredParameter<string>();
 
         /// <summary>
+        /// This specify if the file needs to be copied to local in FilePathsForInputPartitions 
+        /// </summary>
+        public static readonly OptionalParameter<bool> CopyLocal = new OptionalParameter<bool>();
```

Comment:
It would be same as that class name even compiler is able to distinguish them. I will change it.